### PR TITLE
wip

### PR DESCRIPTION
**Pull request by wdsrocha**
#### How to test it?

[Workspace](Link goes here!)

### CODE REVIEW

File: react/OrderItems.tsx [hunk 1]
```diff
@@ -177,12 +177,12 @@ const useEnqueueTask = () => {
         },
         (error: any) => {
           if (error && error.code === constants.TASK_CANCELLED_CODE) {
-            popLocalOrderQueue(error.index)
+            popLocalOrderQueue(error.index, { log: logSplunk })
```
Comment (lucasecdb) [inline react/OrderItems.tsx:180]: why not pass `logSplunk` directly?

Comment (wdsrocha) [reply react/OrderItems.tsx:180]: you mean like
```suggestion
            popLocalOrderQueue(error.index, { logSplunk })
```

or
```suggestion
			popLocalOrderQueue(error.index, logSplunk)
```
?

Comment (lucasecdb) [reply react/OrderItems.tsx:180]: the second one

Comment (wdsrocha) [reply react/OrderItems.tsx:180]: hmmm
my line of thought was that we might need to pass more params in the future, but now that you ask, I realize that this shouldn't matter, as this isn't a public api

Comment (wdsrocha) [reply react/OrderItems.tsx:180]: so sure, I can do that

Comment (lucasecdb) [reply react/OrderItems.tsx:180]: exactly hahah 😄 we can change this anytime if we feel the API is becoming a bit awkward

Comment (wdsrocha) [reply react/OrderItems.tsx:180]: ok, will do like that when implementing on the pkg ✨

**Pull request closed without merging**